.github/workflows/commit-check.yml

```diff
@@ -0,0 +1,34 @@
+name: ForgeRock Commit Check
+on:
+  pull_request:
+  push:
+    branches-ignore:
+      # We don't want to check commits on develop or master
+      # because they will be merged, not the commits we write
+      - 'develop'
+      - 'master'
+
+jobs:
+  commit-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        id: cache
+        with:
+          node-version: '16.13.1'
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+      - name: npm install
```

Comment:
we need npm install as part of this step because it uses a node package for the config. however, this step could be abstracted.